barretenberg/cpp/src/barretenberg/stdlib/recursion/CMakeLists.txt

```diff
@@ -1 +1 @@
-barretenberg_module(stdlib_recursion ecc proof_system stdlib_primitives stdlib_pedersen_commitment stdlib_blake3s ultra_honk eccvm translator_vm stdlib_poseidon2 circuit_checker)
\ No newline at end of file
+barretenberg_module(stdlib_recursion ecc proof_system stdlib_primitives stdlib_pedersen_commitment stdlib_blake3s protogalaxy eccvm translator_vm stdlib_poseidon2 circuit_checker)
```

Comment:
maybe could keep both in here. Is there no impact if we link ultra_honk twice?

Reply:
no impact

Reply:
meh, doesn't seem to matter that much, so I left it unchanged